README.md

```diff
@@ -121,6 +121,7 @@ The following are the flags you can pass to the `dotenv-extended` cli with their
 --schema=.env.schema
 --errorOnMissing=false     # or --error-on-missing=false
 --errorOnExtra=false       # or --error-on-extra=false
+--includeProcessEnv=false       # or --error-check-process=false
```

Comment:
Please update `--error-check-process=false` to be `--include-process-env=false` and align comments